chain/actors/bundle.go

```diff
@@ -0,0 +1,151 @@
+package actors
```

Comment:
nit: I'm not convinced that a component that's doing HTTP requests should be in this package. I think this belongs more in the node package.

Reply:
agreed, i'll move it.